adapters/openweb/openweb.go

```diff
@@ -0,0 +1,193 @@
+package openweb
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/mxmCherry/openrtb/v15/openrtb2"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+type OpenWebAdapter struct {
```

Comment:
This type name should be `adapter`. There is no need to repeat the package name and no reason to export it with an upper case letter. Please review our new adapter docs: https://docs.prebid.org/prebid-server/developers/add-new-bidder-go.html